doc/specs/examples/users-getUserPublicData.json

```diff
@@ -1,124 +1,153 @@
 {
-  "id": "mary",
-  "username": "Mary",
+  "id": "thibault",
+  "username": "thibault",
```

Comment:
ok for this time but let's aim for some variety :) I wouldn't want to be all over the API docs